README.md

```diff
@@ -1,95 +1,11 @@
 
-### Set up the repository
+We recommend that most new users use the code from the stable `master` branch. Please post questions to GitHub issues with the `question` tag. We are prioritizing direct support for individuals engaged in public health planning and emergency response.
 
-**Please see the [Wiki for this repository](https://github.com/HopkinsIDD/COVIDScenarioPipeline/wiki) for updated instructions on how to clone the repository and push/pull changes.**
+For more information on getting started, please visit our [Getting Started wiki](https://github.com/HopkinsIDD/COVID19_Minimal/wiki/Getting-Started) at [HopkinsIDD/COVID19_Minimal](https://github.com/HopkinsIDD/COVID19_Minimal). We are trying to keep this page up-to-date for use with the `master` branch.
 
-If making changes to this repository, please do it directly instead of through the submodule of another repository.
+This is an open-source code licensed under GPL v3.0. 
 
-### Run the code
 
-After cloning the repository (see [Wiki](https://github.com/HopkinsIDD/COVIDScenarioPipeline/wiki) for instructions on how to clone a repository with submodules) create setup in a subfolder of the `data` using the notebook `data/build-model-input.ipynb`. This creates two files:
-
-* `mobility.txt` : mobility matrix from us commute, as in [An Economic Geography of the United States: From Commutes to Megaregions by Garrett Dash Nelson and Alasdair Rae](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0166083) processed by [Ed King on Kaggle](https://www.kaggle.com/kinguistics/visualizing-u-s-commutes)
-* `geodata.csv` : specification of the spatial nodes, with at least column for the index, the geoid or name, the population.
-
-Then copy the `main_template.py` from this repo into the root of the SPATIAL_SETUP_REPO folder. Changes the first line and you're ready to run it.
-
-if you haven't installed python packages, run
-
-```
-pip3 install -r requirements.txt
-```
-
-You can now run the code:
-```
-nohup python3 main.py > out.txt &
-```
-creates a `figure/` and a `SCENARIO_model_output_TIMESTAMP` folder.
-
-
-# Docker
-
-A containerized environment is a packaged environment where all
-dependencies are bundled together. This means you're guaranteed to be
-using the same libraries and system configuration as everyone else and in
-any runtime environment. To learn more, [Docker
-Curriculum](https://docker-curriculum.com/) is a good starting point.
-
-## Starting environment
```

Comment:
I think leaving the stuff in about the docker hub images seems useful.  Otherwise seems good to me

Reply:
check new commit